pkg/lib/authn/user/model.go

```diff
@@ -317,6 +318,36 @@ func newUserModel(
 	}
 }
 
+func newUserModelForExport(
+	user *User,
+	identities []*identity.Info,
+	authenticators []*authenticator.Info,
+	isVerified bool,
+	derivedStandardAttributes map[string]interface{},
+	customAttributes map[string]interface{},
+	web3Info *model.UserWeb3Info,
+	roles []string,
+	groups []string,
+) *userexport.UserForExport {
+	modelUser := newUserModel(
+		user,
+		identities,
+		authenticators,
+		isVerified,
+		derivedStandardAttributes,
+		customAttributes,
+		web3Info,
+		roles,
+		groups,
+	)
+
+	return &userexport.UserForExport{
+		User:           *modelUser,
+		Identities:     identities,
+		Authenticators: authenticators,
+	}
+}
+
```

Comment:
This function should live in pkg/lib/userexport

Reply:
removed this function as it is not necessary